mcc/GetSetAttrs.c

```diff
@@ -453,9 +456,9 @@ IPTR mSet(struct IClass *cl, Object *obj, struct opSet *msg)
                     * data->fontheight,
                   TAG_DONE);
 
-        // visual_y is changed it is best re-calculate the longest visible line here
+/*   // visual_y is changed it is best re-calculate the longest visible line here
```

Comment:
Please remove all commented out code in /* */ if it is really not required anymore. That's exactly what a revision control system is all about.